Channels/Integration/02_ManageVariantSets.md

```diff
@@ -0,0 +1,184 @@
+
+#### Procedure
+
+*Omni-Channel > Settings > Tab VARIANT SETS > Button Add*
+
+![Formulas](../../Assets/Screenshots/Channels/Settings/VariantSets/Formulas.png "[Formulas]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") button in the *Defining attributes* box. This button is only displayed if an attribute set is selected.       
+  The *Add defining attributes* view is displayed.
+
+  ![Add defining attributes](../../Assets/Screenshots/Channels/Settings/VariantSets/AddDefiningAttributes.png "[Add defining attributes]")
+
+2. Select the checkboxes of the attributes you want to be a defining attribute for the selected variant.   
+  The editing toolbar is displayed above the attribute list.
+
+  > [Info] The defining attributes are those attributes that characterize the variant, so those attributes that are defining the variants of the master product.
+
+3. Click the [ADD AND GO BACK] button.     
+    The *Add defining attributes* view is closed. The selected attributes are displayed in the *Defining attributes* box. A placeholder for the added attribute is displayed in the *Formulas* section.   
+
+    ![Defining attributes added](../../Assets/Screenshots/Channels/Settings/VariantSets/DefiningAttributesAdded.png "[Defining attributes added]")
+
+    > [Info] You can edit the formula in the *Formula for SKU* field and include the defining attribute placeholder(s).
+
+4. If desired, add changeable attributes to the variant set, see [Add changeable attributes](#add-changeable-attributes). Otherwise, click the [SAVE] button in the upper right corner.   
+  The *Create variant set* view is closed. The new variant set has been saved and is displayed in the list of variant sets.
+
+
+
+### Add changeable attributes
+
+You can select changeable attributes for a variant set. If you want to define an attribute for the variants whose value may differ from its master product value, you have to define this attribute as a changeable attribute.
+
+#### Prerequisites
+
+- An attribute set has been created in DataHub, see [Create an attribute set](../../DataHub/Integration/02_ManageAttributeSets.md#create-an-attribute-set).
+- The basic data of the variant set has been defined, see [Define the variant set basic data](#define-the-variant-set-basic-data).
+
+#### Procedure
+
+*Omni-Channel > Settings > Tab VARIANT SETS > Button Add*
+
+![Formulas](../../Assets/Screenshots/Channels/Settings/VariantSets/Formulas.png "[Formulas]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") button in the *Changeable attributes* box. This button is only displayed if an attribute set is selected.         
+  The *Add changeable attributes* view is displayed.
+
+  ![Add changeable attributes](../../Assets/Screenshots/Channels/Settings/VariantSets/AddChangeableAttributes.png "[Add changeable attributes]")
+
+2. Select the checkboxes of the attributes you want to be a changeable attribute for the selected variant.   
```

Comment:
```suggestion
2. Select the checkboxes of the attributes you want to be a changeable attribute for the selected variant set.   
```

Reply:
Aus Interesse für mich: Warum nicht "for the selected variant set"? Die Attribute, die man hier auswählt, treffen alle Varianten im Variant set, oder?